--- a/src/Kopernicus/Kopernicus.csproj	2019-05-10 19:19:10.348744438 +0200
+++ b/src/Kopernicus/Kopernicus.csproj	2019-05-10 19:35:28.740053886 +0200
@@ -14,7 +14,7 @@
     <DebugType>full</DebugType>
     <Optimize>False</Optimize>
     <OutputPath>..\..\build\GameData\Kopernicus\Plugins\</OutputPath>
-    <DefineConstants>DEBUG;</DefineConstants>
+    <DefineConstants>DEBUG;{FLAG}</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>False</ConsolePause>
@@ -24,6 +24,7 @@
     <DebugType>none</DebugType>
     <Optimize>True</Optimize>
     <OutputPath>..\..\build\GameData\Kopernicus\Plugins\</OutputPath>
+    <DefineConstants>{FLAG}</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>False</ConsolePause>
@@ -31,11 +32,11 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>..\..\packages\Assembly-CSharp.{BASE}\lib\net35\Assembly-CSharp.dll</HintPath>
+      <HintPath>..\..\packages\Assembly-CSharp.{BACKPORT}\lib\net35\Assembly-CSharp.dll</HintPath>
       <Private>False</Private>
     </Reference>
     <Reference Include="ModularFlightIntegrator, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>..\..\packages\ModularFlightIntegrator.{MFIBASE}\lib\net35\ModularFlightIntegrator.dll</HintPath>
+      <HintPath>..\..\packages\ModularFlightIntegrator.{MFIBACKPORT}\lib\net35\ModularFlightIntegrator.dll</HintPath>
       <Private>False</Private>
     </Reference>
     <Reference Include="System">
@@ -45,11 +46,11 @@
       <Private>False</Private>
     </Reference>
     <Reference Include="UnityEngine, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>..\..\packages\UnityEngine.{BASE}\lib\net35\UnityEngine.dll</HintPath>
+      <HintPath>..\..\packages\UnityEngine.{BACKPORT}\lib\net35\UnityEngine.dll</HintPath>
       <Private>False</Private>
     </Reference>
     <Reference Include="UnityEngine.UI, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>..\..\packages\UnityEngine.UI.{BASE}\lib\net35\UnityEngine.UI.dll</HintPath>
+      <HintPath>..\..\packages\UnityEngine.UI.{BACKPORT}\lib\net35\UnityEngine.UI.dll</HintPath>
       <Private>False</Private>
     </Reference>
   </ItemGroup>