--- a/src/external/config-parser/src/Kopernicus.Parser.csproj	2019-05-10 19:02:30.326439772 +0200
+++ b/src/external/config-parser/src/Kopernicus.Parser.csproj	2019-05-10 19:33:06.752869768 +0200
@@ -17,7 +17,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>DEBUG;TRACE;{FLAG}</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
@@ -25,23 +25,23 @@
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
     <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
+    <DefineConstants>TRACE;{FLAG}</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>..\..\..\..\packages\Assembly-CSharp.{BASE}\lib\net35\Assembly-CSharp.dll</HintPath>
+      <HintPath>..\..\..\..\packages\Assembly-CSharp.{BACKPORT}\lib\net35\Assembly-CSharp.dll</HintPath>
       <Private>False</Private>
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="UnityEngine, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>..\..\..\..\packages\UnityEngine.{BASE}\lib\net35\UnityEngine.dll</HintPath>
+      <HintPath>..\..\..\..\packages\UnityEngine.{BACKPORT}\lib\net35\UnityEngine.dll</HintPath>
       <Private>False</Private>
     </Reference>
     <Reference Include="UnityEngine.UI, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>..\..\..\..\packages\UnityEngine.UI.{BASE}\lib\net35\UnityEngine.UI.dll</HintPath>
+      <HintPath>..\..\..\..\packages\UnityEngine.UI.{BACKPORT}\lib\net35\UnityEngine.UI.dll</HintPath>
       <Private>False</Private>
     </Reference>
   </ItemGroup>